site/pages/create-consequence-network/[disruptionId]/[consequenceIndex].page.tsx

```diff
@@ -82,6 +85,51 @@ const CreateConsequenceNetwork = (props: CreateConsequenceNetworkProps): ReactEl
                             ]}
                         />
 
+                        {!["preprod", "prod"].includes(props.stage || "development") ? (
```

Comment:
If we go with the NETWORK_AREA_FEATURE_FLAG we can reuse it here

Reply:
done